fedimint-core/src/lib.rs

```diff
@@ -109,12 +109,16 @@ pub struct Amount {
 impl Amount {
     pub const ZERO: Self = Self { msats: 0 };
 
-    pub const fn from_msats(msat: u64) -> Amount {
-        Amount { msats: msat }
+    pub const fn from_msats(msats: u64) -> Amount {
+        Amount { msats }
     }
 
-    pub const fn from_sats(sat: u64) -> Amount {
-        Amount { msats: sat * 1000 }
+    pub const fn from_sats(sats: u64) -> Amount {
+        Amount::from_msats(sats * 1000)
+    }
+
+    pub const fn from_coins(coins: u64) -> Amount {
```

Comment:
**suggestion:** No idea what "from_coins" supposed to mean, and only after looks at the impl I see it's "from_btc"

Reply:
I can throw together a PR that replaces this line:

https://github.com/fedimint/fedimint/blob/479bebe32648841c203993eed925462caebd65ce/modules/fedimint-mint-common/src/config.rs#L27

with

`const MAX_DENOMINATION_SIZE: Amount = Amount::from_sats(1_000_000 * 100_000_000);`

and remove `from_coins()`